_posts/2017-07-26-a-web-animations-deep-dive-with-angular.md

```diff
@@ -353,7 +353,7 @@ this.timeline
 {% endraw %}
 {% endhighlight %}
 
-Woah! This looks pretty overwhelming at a first glance. But all we do is is to orchestrate our animation using GreenSock's timeline API. With the help of labels we can run multiple animations in parallel and precisely control the timing of certain animations.
+Woah! This looks pretty overwhelming at first glance. But all we have to do is to orchestrate our animation using GreenSock's timeline API. With the help of labels we can run multiple animations in parallel and precisely control the timing of certain animations.
```

Comment:
I'm not sure whether `have to do` is the correct tense here. But `But all we do is is to` was incorrect and `But all we do is to` sounds odd.